### PR TITLE
Develop branch

### PR DESCRIPTION
**Pull request by sohag16030**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -9,3 +9,4 @@ spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 
+spring.jackson.serialization.fail-on-empty-beans=false
```
Comment (suvro47) [inline src/main/resources/application.properties]: Eta ki kore?

File: src/main/java/com/cms/example/cms/feature/GeoController.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+public class GeoController {
+
+    private final ServiceUtil service;
```
Comment (suvro47) [inline src/main/java/com/cms/example/cms/feature/GeoController.java]: Utils is kind of helper class. In this case its not utils. Rename it as 'GeoService'

**Review by suvro47 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (sohag16030) [reply src/main/resources/application.properties]: serialize krtepartesilo na e jonno ad krsilm.but etao kaj kore ni..remove koew disi

File: src/main/java/com/cms/example/cms/feature/GeoController.java (hunk 2 above)
Comment (sohag16030) [reply src/main/java/com/cms/example/cms/feature/GeoController.java]: updated

**Pull request merged**